swagger.yaml

```diff
@@ -5717,6 +5717,24 @@ components:
           format: uuid
           readOnly: true
           description: Id of ExhibitionDeviceGroup that ExhibitionDevice belongs to
+          
+    MqttDeviceStatus:
+      type: object
+      description: |
+        MQTT notification message about devices status. This is actually not part of the REST API spec but a realtime notification via MQTT channel.
+        It is periodically sent by all devices to the server.
+      properties:
+        deviceId:
+          type: string
+          format: uuid
+          readOnly: true
```

Comment:
This can cause problems when constructing the model, are you sure that is should be readOnly?

Reply:
Resolved.